controller/src/test/java/io/pravega/controller/store/stream/ZKStreamMetadataStoreTest.java

```diff
@@ -213,8 +213,6 @@ private void testFailure(String host, TxnResource txn, Predicate<Throwable> chec
         AssertExtensions.assertFutureThrows("Remove txn fails", store.removeTxnFromIndex(host, txn, true), checker);
         AssertExtensions.assertFutureThrows("Remove host fails", store.removeHostFromIndex(host), checker);
         AssertExtensions.assertFutureThrows("Get txn version fails", store.getTxnVersionFromIndex(host, txn), checker);
-        AssertExtensions.assertFutureThrows("Get random txn fails", store.getRandomTxnFromIndex(host), checker);
```

Comment:
Why are we removing these assertions?

Reply:
These are the assertions that result in getEntities on index being called. GetEntities has `sync` method which times out because curator client is already closed and we do not get the callback invoked by curator for sync method. 
we have also opened an issue in curator's JIRA for this issue (https://issues.apache.org/jira/browse/CURATOR-529)